config/locales/en/tasklist.yml

```diff
@@ -24,3 +24,21 @@ en:
       not_started: Not started
       unreachable: Cannot yet start
       not_applicable: Not applicable
+
+  crime_applications:
```

Comment:
I think the rebase failed with the locales. All this should go to `dashboard.yml` now. Also be careful as I think some of the `edit` action locales are botched.